src/client/src/ui/StatusBar/StatusBar.tsx

```diff
@@ -1,16 +1,27 @@
 import _ from 'lodash'
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import { ThemeProvider } from 'rt-theme'
+import { styled, ThemeProvider } from 'rt-theme'
 
+import { GlobalState } from 'combineReducers'
 import Icon from './Icon'
-import { Root, Body, Fill, ExpandToggle, ServiceList, ServiceRoot, ServiceName, NodeCount } from './styled.js'
+import { Body, ExpandToggle, Fill, NodeCount, Root, ServiceList, ServiceName, ServiceRoot } from './styled'
 
 export const SERVICES = ['blotter', 'reference', 'execution', 'pricing', 'analytics'].map(serviceType => ({
-  serviceType
```

Comment:
You've broken functionality — remove this change.

Or, revisit the use to understand the change you'd like to make.

The correct value for these keys is `null`.

Reply:
What does it break?  They had a value of undefined before this change.

Reply:
isConnected = null | true | false 

makes no sense